app/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout>
```

Comment:
Toàn bộ file này vẫn cần tách string và dimen ra 1 file riêng, anh bổ sung 2 phần này nhé